src/test/java/com/jcabi/dynamo/AttributeUpdatesTest.java

```diff
@@ -0,0 +1,285 @@
+            Matchers.notNullValue()
+        );
+    }
+
+    /**
+     * AttributesUpdates can tell if it contains a key.
+     */
+    @Test
+    public void containsKey() {
+        final String key = "key11";
+        final AttributeUpdates attr = new AttributeUpdates()
+            .with(key, "value11").with("otherkey1", "othervalue1");
+        MatcherAssert.assertThat(
+            attr.containsKey(key),
+            Matchers.is(Boolean.TRUE)
+        );
+    }
+
+    /**
+     * AttributesUpdates can tell if it contains a value.
+     */
+    @Test
+    public void containsValue() {
+        final String value = "attrv";
+        final AttributeUpdates attr = new AttributeUpdates()
+            .with("attrkey", value).with("otherkey", "othervalue");
+        MatcherAssert.assertThat(
+            attr.containsValue(
+                new AttributeValueUpdate(
+                    new AttributeValue(value), AttributeAction.PUT
+                )
+            ),
+            Matchers.is(Boolean.TRUE)
+        );
+    }
+
+    /**
+     * AttributesUpdates can return a string representation of itself.
+     */
+    @Test
+    public void canTurnToString() {
+        final AttributeUpdates attr = new AttributeUpdates()
+            .with("onekey", "onevalue").with("secondkey", "secondvalue");
+        MatcherAssert.assertThat(
+            attr.toString(),
+            Matchers.equalTo(
+                StringUtils.join(
+                    "onekey={Value: {S: onevalue,},Action: PUT}; ",
+                    "secondkey={Value: {S: secondvalue,},Action: PUT}"
+                )
+            )
+        );
+    }
+
+    /**
+     * AttributesUpdates can add the content of another Map.
+     */
+    @Test
+    public void addsMap() {
+        final AttributeUpdates attr = new AttributeUpdates();
+        MatcherAssert.assertThat(attr.size(), Matchers.is(0));
+        MatcherAssert.assertThat(
+            attr.with("testkey8", new AttributeUpdates().with("key", "value"))
+                .size(),
+            Matchers.is(1)
+        );
+    }
+
+    /**
+     * AttributesUpdates can throw exception when put is called.
+     */
+    @Test
+    public void putThrowsException() {
+        try {
+            new AttributeUpdates().put(
+                "key9", Mockito.mock(AttributeValueUpdate.class)
+            );
+            Assert.fail("#put should not be supported");
+        } catch (final UnsupportedOperationException ex) {
+            MatcherAssert.assertThat("passed test", Matchers.notNullValue());
+        }
+    }
+
+    /**
+     * AttributesUpdates can throw exception when putAll is called.
+     */
+    @Test
+    public void putAllThrowsException() {
+        try {
+            new AttributeUpdates().putAll(new AttributeUpdates());
+            Assert.fail("#putAll should not be supported.");
+        } catch (final UnsupportedOperationException ex) {
+            MatcherAssert.assertThat("test ok", Matchers.notNullValue());
```

Comment:
@amihaiemil that assertion is a bit awkward for me. Personally I'd go with `// expected` comment or if comment violates Qulice as well I'd make asssertion on `ex.getMessage` - whatever it is